lib/functional_mixin.js

```diff
@@ -95,6 +108,58 @@ function busterFunctionalMixin(testCase, options)
     return this;
   };
 
+  testCase.swipeLeft = function busterFunctionalMixin_swipeLeft(target, callback)
+  {
+    // update offset value
+    var offset = this.window && this.window.innerWidth ?
+                 this.window.innerWidth * this._swipeRightOrLeftOffsetFactor :
+                 this._swipeRightOrLeftDistance;
+
+    this._swipe(target, [-Math.abs(offset), 0], callback);
+
+    return this;
+
+  };
+
+  testCase.swipeRight = function busterFunctionalMixin_swipeRight(target, callback)
+  {
+    // update offset value
+    var offset = this.window && this.window.innerWidth ?
+                 this.window.innerWidth * this._swipeRightOrLeftOffsetFactor :
+                 this._swipeRightOrLeftDistance;
+
+    this._swipe(target, [Math.abs(offset), 0], callback);
+
+    return this;
+
+  };
+
+  testCase.swipeUp = function busterFunctionalMixin_swipeUp(target, callback)
+  {
+    // update offset value
+    var offset = this.window && this.window.innerHeight ?
+                 this.window.innerHeight * this._swipeUpOrDownOffsetFactor :
+                 this._swipeUpOrDownDistance;
```

Comment:
Vertical scrolling for percentage of the screen height, might be not as obvious as horizontal.
How about if we scroll for number of inches in physical world? :) You know, like 1.5-2 inches, this is how my thumb goes, 2-3 inches if I use index finger. Probably most of the people have similar characteristics. What do you think? :)